File: C/C.W and H.W/divisible_by_2_or_3.c
#include <stdio.h>

int main(void){
	int num;
	printf("Enter number");
	scanf("%d",&num);
	if ((num%2==0) || (num%3==0)){
		if (num%2==0){
			printf("The number is divisible by 2");
		}
		if (num%3==0){
			printf("The number is divisible by 3");
		}
	}
	else{
		printf("The number is neither divisible by 2 or 3");
	}
}

File: C but in python/divisible_by_2_or_3.py
num = int(input("Enter number: "))
if (num%2==0) or (num%3==0):
	if num%2==0:
		print("It is divisible by 2")
	if num%3==0:
		print("It is divisible by 3")
else:
	print("It is neither divisible by 2 or 3")

